client/src/views/PeerReview/PeerReview.js

```diff
@@ -111,11 +111,11 @@ const PeerReview = props => {
   if (peerReviewData) {
     filteredEmployee = peerReviewData.data.data.filter(
       cls =>
-        cls.employee_under_review.firstname +
+        (cls.employee_under_review.firstname +
           ' ' +
-          cls.employee_under_review.lastname
-            .toLowerCase()
-            .includes(selectedEmployee.toLowerCase().trim()) &&
+          cls.employee_under_review.lastname)
```

Comment:
I think we need to use template literal here. Is it possible to use template literal?

Reply:
Did in PR #31